app/components/why-step.jsx

```diff
@@ -0,0 +1,111 @@
+//https://github.com/EnCiv/civil-pursuit/issues/103
+
+'use strict';
+
+import React, { useState, useEffect } from 'react';
+import cx from 'classnames';
+import { createUseStyles } from 'react-jss';
+import WhyInput from './why-input.jsx';
+
+export default function WhyStep(props) {
+    const {
+        className = '',
+        type = '', // "most" or "least"
+        intro = '',
+        shared = { mosts: [], leasts: [], whyMosts: [], whyLeasts: [] },
+        onDone = () => { },
+        ...otherProps
+    } = props;
+    const classes = useStylesFromThemeFunction({ valid: true, value: [] });
```

Comment:
Everything works great! But I just noticed this and I don't know if I missed it before. Usually no args are passed into useStylesFromThemeFuction, though it is possible and sometimes props are passed in, but in this case the code is not using it. But passing this in might be confusing to the next person, so can you just make it `const classes = useStylesFromThemeFunction()`

Reply:
Huh. No idea what I was doing there, but it's fixed.